src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -45,7 +46,11 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
     closeUserProfileModal,
   } = useProfileModalControls();
 
-  const chosenUserId = selectedUserProfile?.id;
+  const chosenUserId = selectedUserProfile?.id ?? "";
```

Comment:
```suggestion
  const chosenUserId = selectedUserProfile?.id;
```

Any reason to have "" as default? It can lead to unexpected behaviours

Reply:
you're right. I need to hide it when it is not defined

Reply:
https://github.com/sparkletown/sparkle/pull/1484/commits/6fb7c069dc5c0c57ab4df3f0cc0c358dfe307793